sotopia/database/serialization.py

```diff
@@ -175,6 +176,25 @@ def get_social_interactions_from_episode(
     return "\n\n".join(overall_social_interaction)
 
 
+def _serialize_data_to_csv(data: dict[str, list[Any]], csv_file_path: str) -> None:
```

Comment:
so when outputting things to csv, we should be extremely careful about situations where there are "," in the string of the entry. I don't see that being handled in this function. Pandas automatically handle that so maybe check what they did there?

Another option is to get rid of the csv output and switch everything to jsonl

Reply:
This was automatically handled by the builtin csv writer.

Reply:
Right, I could be wrong, but there are many previous cases that I remembered csv.writer could get things wrong or make the file hard to read. 

That's primarily why I switched to pandas

Reply:
have you tested it to see whether you can load the CSV data back? and how?

Reply:
Yes, verified in 9729ec4

Reply:
Nice!! Thanks!